lib/hex_web/api/router.ex

```diff
@@ -78,7 +78,12 @@ defmodule HexWeb.API.Router do
         User.delete(user)
       end
 
-      result = User.create(conn.params)
+      result = case User.create(conn.params) do
+        {:ok, user} ->
+          user = HexWeb.Repo.preload(user, :owned_packages)
+          {:ok, user}
+        {:error, changeset} -> {:error, changeset}
+      end
```

Comment:
@ericmj had to do this "hack" because of how ```send_creation_resp``` works (on the line below this). Thoughts?